Zotero/Scenes/Detail/ItemDetail/Views/ItemDetailFieldEditCell.swift

```diff
@@ -51,7 +62,9 @@ final class ItemDetailFieldEditCell: UICollectionViewListCell {
         }
 
         private func apply(configuration: ContentConfiguration) {
-            self.contentView.textChanged = configuration.textChanged
+            let disposable = self.contentView.textObservable.bind(to: configuration.textObservable)
+            _ = configuration.disposeBag.insert(disposable)
```

Comment:
Could this lead to having more than one text observers, if the cell is reused?
Should the `disposeBag` be emptied when the cell is prepared for reuse?

Reply:
I thought that since the dispose bag is stored in `ContentConfiguration` it should be enough, but disposing on `prepareForReuse` definitely won't hurt. See 662d34fb9f18d6dc52cda3f161fa3f554603b6a2